rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingSpec.scala

```diff
@@ -263,6 +264,22 @@ class CostAccountingSpec extends FlatSpec with Matchers with PropertyChecks with
     }
   }
 
+  "replay the same term with not enough phlo" should "get the same result" in {
+    val r = scala.util.Random
+    for (_ <- 1 to 100) {
+      val long        = ((r.nextLong % 0X144000000L) + 0X144000000L) % 0X144000000L
```

Comment:
I'm curious what this is doing?

Reply:
To be very honest, I don't know. :) Maybe it is trying to generate a number with some kind of range.

Reply:
Hm. I think it is useless. I would just remove it.

Reply:
After https://github.com/rchain/rchain/runs/4932655784?check_suite_focus=true, I think these codes are for the `r` is negative situation.